test/typing/plain_files/ext/asyncio/engines.py

```diff
@@ -71,3 +73,25 @@ async def asyncio() -> None:
     ce.statement
     cc = select(1).compile(conn)
     cc.statement
+
+    async with e.connect() as conn:
+        metadata = MetaData()
+
+        await conn.run_sync(metadata.create_all)
+        await conn.run_sync(metadata.reflect)
```

Comment:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5242